scss/_button-group.scss

```diff
@@ -5,6 +5,7 @@
 .btn-group-vertical {
   position: relative;
   display: inline-flex;
+  flex-flow: row wrap;
```

Comment:
The initial value of `flex-direction` is `row`. You just need to change the wrap here, not the directions.

I'd make this `flex-wrap: wrap;` instead.

Reply:
It make sense, PR re-send
https://github.com/twbs/bootstrap/pull/24120